portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -135,7 +135,7 @@
 
     <servlet>
         <servlet-name>query_builder</servlet-name>
-        <servlet-class>org.mskcc.cbio.portal.servlet.QueryBuilder</servlet-class>
```

Comment:
This replaces **entire functionality** of `QueryBuilder` with the new `CommonQueryBuilder`. `QueryBuilder` becomes an obsolete servlet.